### PR TITLE
[BLOG]-add-making-gpus-accessible-to-pydata-ecosystem-via-array-api

### PR DESCRIPTION
**Pull request by noatamir**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

## Links to other Quansight blogs: 
https://labs.quansight.org/blog/2021/11/pydata-extensibility-vision/
no llc links found
☝️ this will be tweaked after the migration of this blog.
New link: https://labs.quansight.org/library/blog/2021/11/pydata-extensibility-vision/

### CODE REVIEW

File: apps/labs/posts/making-gpus-accessible-to-pydata-ecosystem-via-array-api.md [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+  supports the standard.
+- to access all the functions that work with the array object without having
+  the need to import the Python module explicitly.
+
+NEP 47 recommends defining a utility function named `get_namespace` to check
+for such an attribute.
+
+An example of potential implementation of `get_namespace` function is defined as follows:
+
+```python
```
Comment (gabalafou) [inline apps/labs/posts/making-gpus-accessible-to-pydata-ecosystem-via-array-api.md:127]: The code blocks have syntax highlighting, but the highlighting has contrast accessibility issues. Note to self to create GitHub issue about this.

**Review by gabalafou — CHANGES_REQUESTED** [review]
The only things that should be fixed before merging in:

- The AMD logo image at the bottom of the blog post.

We also need to decide whether or not to keep my commit that fixes the heading levels so they are compliant with axe-core accessibility checks.

File: apps/labs/posts/making-gpus-accessible-to-pydata-ecosystem-via-array-api.md [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+
+This demo work was part of a larger project to make PyData libraries work with
+multiple array libraries and thereby run on GPUs. This is a significant
+undertaking, and this work would not have been possible without funding
+from AMD for a team at Quansight Labs.
+
+<p align="left">
+    <img
+     alt="AMD logo."
+     src="/images/sponsors/AMD_E_Blk_RGB.png"
```
[the remark this answers is not in the document]
Comment (noatamir) [reply apps/labs/posts/making-gpus-accessible-to-pydata-ecosystem-via-array-api.md]: fixed

**Review by gabalafou — APPROVED** [review]
Approved. Just FYI, I added some merge commits to this PR's branch in order to make the preview build URL work. Not sure if you want to clean those up (squash merge?) or merge as-is. Whatever works :)

**Comment by gabalafou** [comment]
You can access a rendered version of this blog post on Vercel at the following URL: https://quansight-labs-published-e4brof80w-quansight.vercel.app/blog/making-gpus-accessible-to-pydata-ecosystem-via-array-api

**Pull request merged**